### DISCUSSION TITLE
Does anyone Know where i can download the GPU 3D Acceleration drivers (iOS)

### DISCUSSION THREAD

**Discussion by stossy11** [utmapp/UTM#4860] [UTM Help (iOS)]
I’m trying to install Linux on my VM and i would like to download the 3D acceleration drivers

iPad Air 5
iPadOS 16.1
UTM 4.1.3

**Comment by Maxfusion26** [comment]
Nowhere for nowadays....((((

**Comment by Andy312432** [comment]
In Linux, it's built into the kernel
